torchreid/engine/engine.py

```diff
@@ -433,9 +430,6 @@ def run(
                     if should_exit:
                         if compression_ctrl.compression_stage == CompressionStage.FULLY_COMPRESSED:
```

Comment:
when the normal training is going, this method returns true?

Reply:
No, I've checked if compression_ctrl is not None for now, please check